server/datastore/mysql/migrations/tables/20210819131107_AddCascadeToHostSoftware.go

```diff
@@ -0,0 +1,46 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20210819131107, Down_20210819131107)
+}
+
+func Up_20210819131107(tx *sql.Tx) error {
+	referencedTables := map[string]struct{}{"hosts": {}, "software": {}}
+	table := "host_software"
+
+	constraints, err := constraintsForTable(tx, table, referencedTables)
+	if err != nil {
+		return err
+	}
+
+	for _, constraint := range constraints {
+		_, err = tx.Exec(fmt.Sprintf(`ALTER TABLE host_software DROP FOREIGN KEY %s;`, constraint))
```

Comment:
can you add `IF EXISTS`? so you don't have to check explicit errors? not sure if that's supported

Reply:
There doesn't seem to be an `IF EXISTS`.

Reply:
confusing MySQL with Postgresql syntax then. My mistake!